MapboxNavigation/RouteMapViewController.swift

```diff
@@ -395,124 +320,148 @@ class RouteMapViewController: UIViewController {
             self.currentStepIndexMapped = routeProgress.currentLegProgress.stepIndex
         }
 
-        if self.annotatesSpokenInstructions {
-            self.mapView.showVoiceInstructionsOnMap(route: self.routeController.routeProgress.route)
+        if self.annotatesSpokenInstructions, let routeController {
+            self.mapView.showVoiceInstructionsOnMap(route: routeController.routeProgress.route)
         }
     }
 
-    var contentInsets: UIEdgeInsets {
-        let top = self.navigationView.instructionsBannerContentView.bounds.height
-        let bottom = self.navigationView.bottomBannerView.bounds.height
-        return UIEdgeInsets(top: top, left: 0, bottom: bottom, right: 0)
-    }
-
-    // MARK: End Of Route
-
-    func embedEndOfRoute() {
-        let endOfRoute = self.endOfRouteViewController
-        addChild(endOfRoute)
-        self.navigationView.endOfRouteView = endOfRoute.view
-        self.navigationView.constrainEndOfRoute()
-        endOfRoute.didMove(toParent: self)
+    /**
+     Updates the current road name label to reflect the road on which the user is currently traveling.
 
-        endOfRoute.dismissHandler = { [weak self] _, _ in
-            self?.routeController.endNavigation()
-            self?.delegate?.mapViewControllerDidDismiss(self!, byCanceling: false)
+     - parameter location: The user’s current location.
+     */
+    func labelCurrentRoad(at rawLocation: CLLocation, for snappedLoction: CLLocation? = nil) {
```

Comment:
Did anything change here or was this simply moved?

Reply:
just moved